artilleryio/artillery
# feat(test): add test for early Fargate task termination

## Description
Test for early Fargate task termination


Flow:
- Trigger Artillery Fargate test run with `--record` and `--output`
- Wait for test to start and get task id from Cloud API
- Stop the task
- Wait for 60sec to see if the process will exit early
- Run assertions:
    - Exited early
    - Exited with code 7
    - Logged Summary report
    - Generates the report file 

## Pre-merge checklist
- [ ] Does this require an update to the docs?No
- [ ] Does this require a changelog entry?No
